.github/workflows/pull_request.yml

```diff
@@ -43,9 +43,6 @@ jobs:
         with:
           path: ${{ steps.get-yarn-cache.outputs.dir }}
           key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
-      - name: Install yarn deps
-        if: steps.cache-yarn.outputs.cache-hit != 'true'
-        run: yarn
```

Comment:
Why was this removed?. If we clear cache of Actions, this would break the actions jobs.

Reply:
Reverted now, thanks.